### PR TITLE
Test/common date setter

### PR DESCRIPTION
**Pull request by MadayAlcala**

### CODE REVIEW

File: src/test/java/com/fundation/logic/common/DateSetterTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ * @version 1.0
+ */
+public class DateSetterTest {
+
+    /**
+     * This unit test verify the setStartOfDay.
+     */
+    @Test
+    public void DateStarTest(){
+        Date dateStart = new Date(2019, 05,02, 05, 20, 22);
```
Comment (aburgosu) [inline src/test/java/com/fundation/logic/common/DateSetterTest.java]: Put spaces between parameters or delete others.

File: src/test/java/com/fundation/logic/common/DateSetterTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        Date actual = DateSetter.setStartOfDay(dateStart);
+        Assert.assertEquals(expected,actual);
+    }
+
+    /**
+     * This unit test verify the setEndOfDay.
+     */
+    @Test
+    public void EndOfDayTest(){
+        Date dateEnd = new Date(2019, 05,02, 05, 20, 22);
```
Comment (aburgosu) [inline src/test/java/com/fundation/logic/common/DateSetterTest.java]: Fix spaces between parameters.

Comment (MadayAlcala) [reply src/test/java/com/fundation/logic/common/DateSetterTest.java]: Done.

File: src/test/java/com/fundation/logic/common/DateSetterTest.java (hunk 1 above)
Comment (MadayAlcala) [reply src/test/java/com/fundation/logic/common/DateSetterTest.java]: Done.

**Pull request merged**